src/Aspire.Hosting.Azure/AzureEnvironmentResource.cs

```diff
@@ -17,7 +19,7 @@ namespace Aspire.Hosting.Azure;
 /// Emits a <c>main.bicep</c> that aggregates all provisionable resources.
 /// </summary>
 [Experimental("ASPIREAZURE001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
-public sealed class AzureEnvironmentResource : Resource
+public sealed class AzureEnvironmentResource : AzureBicepResource
```

Comment:
`AzureEnvironmentResource` needs to implement `AzureBicepResource` so we can set parameters and store outputs from the ARM deployment, like the ACR instance associated with a container apps environment for future image pushes.

Reply:
So this represents main.bicep now? Should this be an AzureProvisoningResource instead?